tests/UKHO.ERPFacade.API.UnitTests/Handlers/S100EventHandlerTests.cs

```diff
@@ -110,5 +111,64 @@ public void WhenSapRespondsWith401Unauthorized_ThenS100EventHandlerRaiseAnExcept
 
             A.CallTo(() => _fakeAzureTableReaderWriter.UpdateEntityAsync(A<string>.Ignored, A<string>.Ignored, A<Dictionary<string, object>>.Ignored)).MustNotHaveHappened();
         }
+
+        [Test]
+        public void WhenValidEventReceivedWithNoSAPAction_ThenS100EventHandlerPublishTheUnitOfsaleUpdatedEventToEesSuccessfully()
```

Comment:
Asserts for PostEventData and UpdateEntityAsync is missing.

Reply:
resolved as suggested.